text/87_interacting_with_stakeholders.md

```diff
@@ -150,7 +150,7 @@ If possible, take a moment to step back and think about how the requirement work
 
 If you still have questions or think you have found some ambiguity, you should discuss the problem with the relevant stakeholders.  These stakeholders will vary based on the domain of the software you're working on and the kind of team you're working with.  In many large companies, systems engineers are in charge of requirements definitions and can help you understand what they were trying to convey.  If it is a design issue, perhaps the UI/UX designers can help you.  If you are working on a startup or a very small team, you may even ask users directly what the think the software should do under those circumstances.
 
-When seeking to clarify requirements, keep in mind that your job as a tester is to objectively test that the system meets the requirements, not to make them!  That means that in general, you should assume that other stakeholders have a better idea of what the expected behavior of a system should be.  This does not mean that you should ignore your thoughts on the matter, or stand idly by while people let you know that "of _course_ the calculator subsystem should crash if anybody enters a number greater than 17."  You should respectfully disagree and tell people your opinions and what you've discovered from a technical perspective.  This is part of being a professional.  Other stakeholders are professional, however, and those whose goals focus on _generating_ requirements instead of _testing_ them will often be a more authoritative source for answers.
+When seeking to clarify requirements, keep in mind that your job as a tester is to objectively test that the system meets the requirements, not to make them!  That means that in general, you should assume that other stakeholders have a better idea of what the expected behavior of a system should be.  This does not mean that you should ignore your thoughts on the matter, or stand idly by while people let you know that "of _course_ the calculator subsystem should crash if anybody enters a number greater than 17."  You should respectfully disagree and tell people your opinions and what you've discovered from a technical perspective.  This is part of being a professional.  However, other stakeholders are professional too, and those whose goals focus on _generating_ requirements instead of _testing_ them will often be a more authoritative source for answers.
```

Comment:
I meant to say something like "professional developers of requirements" but it got all messed up.  I'll merge this and immediately make some more changes to it.